return function(err)
	warn(tostring(err))
	print(debug.traceback())
end
